### ISSUE TITLE
Abgeleitete Entitäten

### ISSUE THREAD

**Issue by maximilian-zollbrecht** [xdoo/vaadin-demo#225]
In  Barraḱuda ist es möglich eine Entität von einer anderen Entität abzuleiten.
Es muss festgelegt werden, wie das genaue Verhalten von GUI und Service in diesem Fall sein soll.
- Anfrage an BasisEntität liefert auch ableitende Entitäten?
- Kann man einer Relation auf die BasisEntität eine abgeleitete Entität zuweisen?

Ich weiß nicht, ob es Sinn macht das direkt ins Referenzsystem einzubauen, weil man dann eine neue Entität bräuchte. Eventuell ist es sinnvoller das direkt zu generieren.

Soll das Im Referenzsystem gebaut werden, oder in Barrakuda?

**Comment by Arne2** [comment]
Ja, der Zweck des Referenzsystems ist auch, dass wir für Möglichkeiten der Generation in Barrakuda auch Beispiele Implementieren. Unwichtig ist, welche abgeleitete Entität wir einbauen.

**Comment by maximilian-zollbrecht** [comment]
Wird mit martkhm/barrakuda#148 erledigt.

**Comment by FabianWilms** [comment]
@maximilian-zollbrecht  Wie sieht es hier mit den Komponenten aus? Sollen wir bspw. Die Grid mit dem Dropdown-Menü zum erstellen generieren, oder ist das Sache des nutzers?
